src/VirtoCommerce.CatalogModule.Core/Model/Property.cs

```diff
@@ -84,6 +84,15 @@ public virtual bool IsSuitableForValue(PropertyValue propValue)
             return string.Equals(Name, propValue.PropertyName, StringComparison.InvariantCultureIgnoreCase) && ValueType == propValue.ValueType;
         }
 
+        public override bool Equals(object obj)
```

Comment:
I would like to recommend to avoid override Equals for Property type in order to avoid side effects that this overriding  may entail

Reply:
I think this code also redundant and can be opted-out